turbo_sort.py

```diff
@@ -231,7 +233,9 @@ def populate_fields(filepath):
 
     # If the filename is encoded like a hashname containing hexadecimal, use the directory name instead
     if re.match(hexaPattern, filename):
-        filename = os.path.basename(os.path.dirname(filepath))
+        # To avoid file erasing, we put the file datetime in the name
+        fileatime = datetime.datetime.fromtimestamp(os.path.getatime(filepath))
+        filename = os.path.basename(os.path.dirname(filepath)) + "_" + fileatime.strftime("%Y%m%d_%H%M%S")
```

Comment:
Isn't this unnecessary with the other fix? Or are you saying that you could have multiple files all with hashed filenames?